TestFlightFailure_IgnitionFail.cs

```diff
@@ -148,6 +153,9 @@ public override float DoRepair()
                 EngineHandler engine = engines[i];
                 {
                     engine.engine.Shutdown();
```

Comment:
As I recall it wasn't so much causing the engine to fail upon repair, but just shut it down in case it was already running.  It is a bit hazy but I seem to recall a situation where you could ignite the engine, get a failure and then if you repaired the failure the engine would immediately jump back into the running state.  So this was just to ensure that if it failed to ignite, and then you fixed it, you would have to re-ignite it using up an ignition.